src/OrchardCore.Themes/TheTheme/Views/MenuItem.cshtml

```diff
@@ -2,22 +2,38 @@
     int level = (int)Model.Level;
     TagBuilder tag = Tag(Model, "li");
 
+    TagBuilder parentTag = null;
     if ((bool)Model.HasItems)
     {
         tag.AddCssClass("dropdown");
+        parentTag = Tag(Model, "ul");
+        parentTag.AddCssClass("dropdown-menu");
     }
 
     // Morphing the shape to keep Model untouched
+    var linkModel = Model.Clone();
```

Comment:
what is this used for?

Reply:
It was an experiment I should have removed.